src/main/perf/LineFileDocs.java

```diff
@@ -698,6 +700,23 @@ public Document nextDoc(DocState doc, boolean expected) throws IOException {
     doc.title.setStringValue(title);
     doc.randomLabel.setStringValue(randomLabel);
     if (addDVFields) {
+      doc.doc.removeField("quarter");
+      doc.doc.removeField("quarterBDV");
+      int randomInt = RANDOM.nextInt(13);
```

Comment:
isn't this used by multiple threads? do we really want them synchronizing on the same random instance?

Reply:
Thanks @rmuir for the review! I was focusing on generating `quarter` field randomly earlier, and did not realize this is used by multiple threads, but it makes sense to not have threads contention here. I have removed this and related changes to use `month` for sorting instead per suggestion from  https://github.com/apache/lucene/pull/12194#issuecomment-1475794788.